.env.hoodi

```
@@ -6,7 +6,7 @@ export REACT_APP_TESTNET_LAUNCHPAD_NAME=Hoodi
 export REACT_APP_GENESIS_FORK_VERSION=0x10000910
 export REACT_APP_RPC_URL=https://rpc.hoodi.ethpandaops.io
 export REACT_APP_BEACONCHAIN_URL=https://dora.hoodi.ethpandaops.io
-export REACT_APP_EL_EXPLORER_URL=https://explorer.hoodi.ethpandaops.io
+export REACT_APP_EL_EXPLORER_URL=https://hoodi.etherscan.io
```

> We could also use https://hoodi.cloud.blockscout.com/ as an alternative EL explorer.

@remyroy Oh great! Honestly would advocate for this... I didn't know about their "cloud" URL...

```suggestion
export REACT_APP_EL_EXPLORER_URL=https://hoodi.cloud.blockscout.com
```